action/protocol/poll/protocol.go

```diff
@@ -45,6 +46,9 @@ type GetEpochHeight func(uint64) uint64
 // GetEpochNum defines a function to get epoch number given a block height
 type GetEpochNum func(uint64) uint64
 
+// InitTryIntervalCtxKey is for InitTryInterval context
+type InitTryIntervalCtxKey struct{}
```

Comment:
You can set the interval when constructing the protocol

No need to pass it over context

Reply:
sure